src/os/bluestore/BlueStore.cc

```diff
@@ -7883,10 +7883,11 @@ void BlueStore::_txc_calc_cost(TransContext *txc)
   // a configurable (with different hdd and ssd defaults), and add
   // that to the bytes value.
   int ios = 1;  // one "io" for the kv commit
+#ifdef HAVE_LIBAIO
   for (auto& p : txc->ioc.pending_aios) {
     ios += p.iov.size();
   }
-
+#endif
```

Comment:
We could put this (and the next hunk) in an IOContext method to avoid HAVE_LIBAIO appearing in BlueStore.cc